Index: src/components/Sample/Sample.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React, { FC } from 'react';\r\nimport { useAppSelector, useAppDispatch } from '../../redux/hooks';\r\nimport { increment, decrement } from '../../redux/reducers/sampleSlice';\r\nimport type { RootState } from '../../redux/store';\r\nimport { TitleStyled } from './Sample.styles';\r\nimport { useTranslation } from 'react-i18next';\r\nimport GoogleSignUp from '../GoogleAuth/GoogleSignUp';\r\n\r\nconst Sample: FC = () => {\r\n  const value = useAppSelector((state: RootState) => state.sample.value);\r\n  const dispatch = useAppDispatch();\r\n  const { t } = useTranslation();\r\n\r\n  return (\r\n    <div>\r\n      <TitleStyled>{`${t('Sample.title')}`}</TitleStyled>\r\n      <div>{value}</div>\r\n      <GoogleSignUp />\r\n\r\n      <button onClick={() => dispatch(decrement())}>-10</button>\r\n      <button onClick={() => dispatch(increment())}>+10</button>\r\n    </div>\r\n  );\r\n};\r\n\r\nexport default Sample;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/Sample/Sample.tsx b/src/components/Sample/Sample.tsx
--- a/src/components/Sample/Sample.tsx	(revision 72a1adf23042d37d73728b0e5a3e229d2f48f17f)
+++ b/src/components/Sample/Sample.tsx	(date 1660155431654)
@@ -15,6 +15,7 @@
     <div>
       <TitleStyled>{`${t('Sample.title')}`}</TitleStyled>
       <div>{value}</div>
+
       <GoogleSignUp />
 
       <button onClick={() => dispatch(decrement())}>-10</button>
