Index: .eslintcache
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>[{\"C:\\\\IT\\\\intership\\\\Devs_Heads_fe\\\\src\\\\localization\\\\i18n.js\":\"1\"},{\"size\":250,\"mtime\":1660557393103}]
===================================================================
diff --git a/.eslintcache b/.eslintcache
--- a/.eslintcache	
+++ b/.eslintcache	
@@ -1,1 +1,1 @@
-[{"C:\\IT\\intership\\Devs_Heads_fe\\src\\localization\\i18n.js":"1"},{"size":250,"mtime":1660557393103}]
\ No newline at end of file
+[{"C:\\IT\\intership\\Devs_Heads_fe\\src\\localization\\i18n.js":"1"},{"size":244,"mtime":1660557535298,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","suppressedMessages":"6","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"k2nv9f","C:\\IT\\intership\\Devs_Heads_fe\\src\\localization\\i18n.js",[],[]]
\ No newline at end of file
